kythe/go/platform/tools/kzip/createcmd/createcmd.go

```diff
@@ -90,21 +97,26 @@ func (c *createCommand) Execute(ctx context.Context, fs *flag.FlagSet, _ ...inte
 		SourceFile: []string{c.source},
 	}
 
+	err = unmarshalDetails(c.details, &cu.Details)
```

Comment:
`if err := unmarshalDetails(c.details, &cu.Details); err != nil {

Reply:
with the below change, this becomes: `details, err := ...` but otherwise remains the same to keep lint happy.